lib/exercises/arrow_functions_and_this/problem.md

```diff
@@ -1,8 +1,8 @@
 Arrow functions help to make the confounding issue of `this` less confounding.
 
-Arrow functions lexically bind the value of `this`. It is very likely that you do not know what that sentence means. And that's OK.
+Arrow functions lexically bind the value of `this`.  It is very likely that you do not know what that sentence means. And that's OK.
```

Comment:
Really?  I find them to be **so** much more readable in monospace fonts, clearly delimiting sentences.

Ah well, then again, your house…  Will do in the morning, along with the wrap fixes I missed.